packages/nx-plugin/src/index.ts

```diff
@@ -1,4 +1,7 @@
-export { InitGeneratorSchema } from './generators/init/schema';
-export { initGenerator } from './generators/init/generator';
+export * from './internal/versions';
+export { type InitGeneratorSchema } from './generators/init/schema';
+export { initGenerator, initSchematic } from './generators/init/generator';
 export { ConfigurationGeneratorOptions } from './generators/configuration/schema';
 export { configurationGenerator } from './generators/configuration/generator';
+export { executeProcess, ProcessConfig } from './internal/execute-process';
+export { objectToCliArgs } from './executors/internal/cli';
```

Comment:
Exporting files from `internal` folder seems like an anti-pattern :raised_eyebrow: 

Some of these exports are just copies from `utils` package (e.g. `executeProcess`, `objectToCliArgs`), why not have `create-cli` import them from `utils`. 

I see you have `create-cli` configured as CommonJS, I'd suggest changing it to ESM. Then it can import both CJS and ESM.

Reply:
Thought it has to be CJS... Will change it.

Reply:
Update:
I changed to CJS and it seems the imports don't work anymore. In the current state you should be able to adjust tsconfig and reproduce it.

Reply:
If you change `create-cli` to ESM and import directly from `utils`, then it works just fine. Got it working in 263e2e7bdcdea6adb0cf684ec7a23ff63ada94d7, along with some E2E test fixes (see [CI statuses](https://github.com/code-pushup/cli/actions/runs/10384678810/job/28752220308)).

Reply:
Thanks a lot!!